crates/jstz_cli/src/term/emoji.rs

```diff
@@ -0,0 +1,4 @@
+use console::Emoji;
+
+pub static WARN: Emoji = Emoji("▲", "");
+pub static ERROR: Emoji = Emoji("✕", "");
```

Comment:
same for errot

Reply:
Ditto for the cross, works well with ANSI colours